content/en/project/_index.md

```diff
@@ -10,17 +10,19 @@ aliases:
 ---
 ## What is the Medley Interlisp Project?
 
-The Medley Interlisp Project was created to revive Medley Interlisp.
+The Medley Interlisp Project was created to revive the "Interlisp experience" by creating an environment that could be used by (and might even be useful for) modern users. Starting from the 20-year-old sources for the not-quite-released Medley 3.5 version of Interlisp, we have made good progress adapting Medley to work in the current environment -- open source, GitHub integration, improving Common Lisp support, running on a wide variety of platforms -- but there is much more to be done.
 
-Because Interlisp has been around a long time, with multiple iterations, you might find the multitude of names for the Interlisp language and related companies confusing. To assist, we have created a [glossary of terms](glossary) used on this website.
+Because of the long history of naming and renaming of groups, software, environments, machines, languages, we have created a [glossary of terms](glossary).
 
-## What are the origins of Interlisp? Why do we care?
+## What are the origins of Medley and Interlisp?
 
-The 1970s and 80s saw major advances in computing and Human-Computer interaction.  Interlisp was one of the many experimental systems developed during these heady times.  It represents an early example of a system with a GUI (Graphical User Interface), and IDE (Integrated Development Environment).
+The 1970s and 80s saw major advances in computing and Human-Computer interaction.  At Xerox PARC there were four IDEs developed with different approaches: Smalltalk, with an object-oriented system; Interlisp, for researchers in AI and exploratory development; Cedar/Mesa, with a strongly typed language for software engineering. They each had a different model for source code management, development and versioning. In common they shared an infrastructure and vision computing and distributed systems, with support for Ethernet networking, printing, file servers and network protocols. Each environment had its own Virtual Machine instruction set tuned for the language "all the way down", in a single address space, in a way that opened the systems to customization.
```

Comment:
three IDEs developed? Or are you counting Cedar and Mesa as distinctly different.